db/seeds/users.rb

```diff
@@ -0,0 +1,44 @@
+User.delete_all
+SlackCredential.delete_all
+LdapCredential.delete_all
+GroupUser.delete_all
+
+slack_user_id = ENV["SLACK_USER_ID"] || 'U03AA0BC0'
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.